PROVESFlightControllerReference/test/int/rtc_test.py

```
@@ -412,11 +418,22 @@ def test_10_double_set_test(fprime_test_api: IntegrationTestAPI, start_gds):
 
     # Double set the alarm
-    alarm_time = datetime.now(timezone.utc) + timedelta(seconds=5)
+    alarm_time = datetime.now(timezone.utc) + timedelta(seconds=60)
+    alarm_time_data = dict(
+        Year=alarm_time.year,
+        Month=alarm_time.month,
+        Day=alarm_time.day,
+        Hour=alarm_time.hour,
+        Minute=alarm_time.minute,
+        Second=alarm_time.second,
+    )
```

Ruff (0.15.14)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)